docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -652,6 +652,46 @@ V&V Checks:
     - 100,000
     - Baseline and alternative scenarios 2, 3, and 4
     - Locations include Pakistan, Nigeria, and Ethiopia. 10 seeds * 10,000 simulants = 100,000 total population.
+  * - 4.7
+    - Correct pregnancy duration for partial term pregnancies
+    - Incomplete
+    - N/A
+    - 10
+    - 100,000
+    - Baseline and alternative scenarios 2, 3, and 4
+    - Locations include Pakistan, Nigeria, and Ethiopia. 10 seeds * 10,000 simulants = 100,000 total population.
+  * - 5.0
+    - Wave I neonatal antibiotics with scale-up scenarios 
+    - Complete
+    - N/A
+    - 10
+    - 100,000
+    - Baseline and alternative scenarios 2 - 7 
+    - Locations include Pakistan, Nigeria, and Ethiopia. 10 seeds * 10,000 simulants = 100,000 total population.
+  * - 6.0
+    - Wave I azithromycin 
+    - Incomplete
+    - N/A
+    - 10
+    - 100,000
+    - Baseline
+    - Locations include Pakistan, Nigeria, and Ethiopia. 10 seeds * 10,000 simulants = 100,000 total population.
+  * - 7.0
+    - Wave I AI ultrasound
```

Comment:
I think this should be misoprostol? We already have AI ultrasound but need misoprostol still.